indexer/src/main/java/au/org/aodn/esindexer/utils/LinkUtils.java

```diff
@@ -0,0 +1,48 @@
+package au.org.aodn.esindexer.utils;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Protocols here are referenced from old portal project(Grails). Currently, didn't
+ * implement all of them. Others can be added if needed.
+ */
+@SuppressWarnings("unused")
+public class LinkUtils {
+
+    // protocols
+    private final static List<String> WMS = Arrays.asList(
```

Comment:
Will regex match better to avoid diff version?

Reply:
Thanks. I will check with Nat to know whether new WMS & WFS will add in the future, and if so, what kind of regex i should use.

Reply:
I doubt she know but from the link WWW:DOWNLOAD-1.0-http--download the 1.0 is version and the other similar x.y.z